data/src/main/java/com/mashup/data/source/remote/dto/responsebody/chat/ChatRoomPagingResponse.kt

```diff
@@ -10,11 +10,11 @@ import com.squareup.moshi.JsonClass
 data class ChatRoomPagingResponse(
     val pageLength: Int,
     val totalPage: Int,
-    val room: List<RoomResponse>,
+    val list: List<RoomResponse>,
```

Comment:
변수 네이밍을 room에서 list로 변경하신 이유가 있나용~~ room이 좀 더 직관적이어보이긴 하는뎁 ㅎㅎㅎ

Reply:
list로 주더라고요 제맘대로 room썼다가 파싱안댐

Reply:
아핫 고렇다면 아래처럼 할 수 있습니당!
```suggestion
    @field:Json(name="list") val room: List<RoomResponse>,
```

Reply:
감사링~~